### DISCUSSION TITLE
Cüzdan kurtarma sıkıntı

### DISCUSSION THREAD

**Discussion by asikral27** [IBAX-io/go-ibax#126] [Bug Submission]
İbax cüzdanı kelimeleri ve sayıları kaybeden kopyalamayan hesaba erişemiyor bunun bi cözüme kavuşturulması gerek cüzdanda bu sayıları kelimeleri bulacak alan yok direk şifre ile girilmesi gerek bu kod işi çok Sacma kaybedince birdaha hesaba ulaşılmama durumu var bunun düzeltilmesi gerek

**Comment by cartvecurt** [comment]
Yeni cüzdan açıp mail yoluyla bağlı olan cüzdanı değiştirmeyi talep edebilirsin belki. Bu durum bütün cüzdanlarda var, yenisini alırken kaybetme o sayıları.
